src/main/java/io/cdap/plugin/gcp/dataplex/common/config/DataplexBaseConfig.java

```diff
@@ -76,6 +78,12 @@ public String getReferenceName() {
     return referenceName;
   }
 
+  public String getReferenceNameOrNoarmalizedFQN(String fqn) {
+    return Strings.isNullOrEmpty(referenceName)
+      ? ReferenceNames.normalizeFqn(fqn)
```

Comment:
nit - can you put this on the same line?

Reply:
+1